webapi/scripts/graphdb-knora-test-data.expect

```diff
@@ -24,14 +24,10 @@ send "load ../_test_data/all_data/system-data.ttl into http://www.knora.org/data
 expect $prompt
 send "load ../_test_data/ontologies/incunabula-onto.ttl into http://www.knora.org/ontology/0803/incunabula .\r"
 expect $prompt
-send "load ../_test_data/all_data/incunabula-data.ttl into http://www.knora.org/data/0803/incunabula .\r"
```

Comment:
Any reason why you would need to remove the test data?

Reply:
damned... no, that's local config committed inadvertently, fixing this